src/Hosting/Server.IntegrationTesting/src/Common/DotNetCommands.cs

```diff
@@ -16,8 +16,13 @@ public static class DotNetCommands
         // Compare to https://github.com/aspnet/BuildTools/blob/314c98e4533217a841ff9767bb38e144eb6c93e4/tools/KoreBuild.Console/Commands/CommandContext.cs#L76
         public static string GetDotNetHome()
         {
+            // runtest.* scripts throughout the repo define $env:DOTNET_HOME
```

Comment:
@jkotalik the changes to this file are an attempt to avoid defining `$env:DOTNET_HOME` globally (see https://github.com/aspnet/AspNetCore/pull/13040/files#diff-f3d39efe7528a28e4995c18938c8a3d3L117) when it has _no_ impact outside 3 or 4 of our tests. But, I'll revert this if this file is actually shipped. (No need for unnecessary product changes at this point in 3.0.)

How is this file used outside our own tests?

Reply:
It shouldn't be used outside of our tests. It's in the integration testing project.